CHANGES.rst

```diff
@@ -5,7 +5,7 @@ Changes to be released in next version
  * 
 
 🙌 Improvements
- * 
```

Comment:
Is there any related issue in https://github.com/vector-im/element-ios/milestone/40?